node/core/approval-voting/src/approval_db/v1/mod.rs

```diff
@@ -35,6 +35,9 @@ use crate::{
 };
 
 const STORED_BLOCKS_KEY: &[u8] = b"Approvals_StoredBlocks";
+const BLOCK_ENTRY_PREFIX: [u8; 14] = *b"Approvals_blck";
+const CANDIDATE_ENTRY_PREFIX: [u8; 14] = *b"Approvals_cand";
+const BLOCKS_AT_HEIGHT_PREFIX: [u8; 12] = *b"Approvals_at";
```

Comment:
nit: Casing is a bit inconsistent, do we really save much by abbreviating? If so, abbreviate more, if not, donate a few extra chars.

Reply:
edit: nvm, these were just collected, ignore the above. Not worth breaking compat.